test-site/.gitignore

```diff
@@ -1,15 +1,17 @@
+public/*
```

Comment:
for *reasons* apparently git is finicky and whether or not you put a star at the end of a path matters
https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder